### DISCUSSION TITLE
rtings.com首页导航栏的图标显示为一个方框

### DISCUSSION THREAD

**Discussion by jinuzukt** [F9y4ng/GreasyFork-Scripts#404] [问答专区(Question / Answer)] [answered]
使用的浏览器：chrome v.131.0.6778.109
脚本管理器：暴力猴 2.29
脚本版本：2024.11.02.1
发现渲染问题的具体网站地址：https://www.rtings.com/

问题：首页导航栏第一个图标显示为方框，对比图一并附上
照猫画虎在rendered elements框内加入:not(::before)以后显示正常了
![rtings 图标渲染问题](https://github.com/user-attachments/assets/0d83d6da-ad06-4172-b197-4483c31c1a0f)
![rtings 图标渲染问题2](https://github.com/user-attachments/assets/43136643-be82-4e66-8c4f-a11054156019)

**Comment by F9y4ng** [comment, marked as answer]
在**需要渲染的网页元素**中，追加`:not(.t-icon,[class^=i-],[class*=" i-"])`，保存为站点独享。

> 你所填写的内容是造成渲染样式无效后，让你感觉到恢复正常了。

**Reply by jinuzukt** [reply]
感谢回复
